doc/manual/local.mk

```diff
@@ -4,83 +4,119 @@ MANUAL_SRCS := \
   $(call rwildcard, $(d)/src, *.md) \
   $(call rwildcard, $(d)/src, */*.md)
 
-# Generate man pages.
 man-pages := $(foreach n, \
-  nix-env.1 nix-build.1 nix-shell.1 nix-store.1 nix-instantiate.1 \
+  nix-env.1 nix-store.1 \
+  nix-build.1 nix-shell.1 nix-instantiate.1 \
   nix-collect-garbage.1 \
   nix-prefetch-url.1 nix-channel.1 \
   nix-hash.1 nix-copy-closure.1 \
-  nix.conf.5 nix-daemon.8, \
-  $(d)/$(n))
+  nix.conf.5 nix-daemon.8 \
+, $(d)/$(n))
+
+# man pages for subcommands
+# convert from `$(d)/src/command-ref/nix-{1}/{2}.md` to `$(d)/nix-{1}-{2}.1`
+# FIXME: unify with how nix3-cli man pages are generated
+man-pages += $(foreach subcommand, \
+  $(filter-out %opt-common.md %env-common.md, $(wildcard $(d)/src/command-ref/nix-*/*.md)), \
+  $(d)/$(subst /,-,$(subst $(d)/src/command-ref/,,$(subst .md,.1,$(subcommand)))))
 
 clean-files += $(d)/*.1 $(d)/*.5 $(d)/*.8
 
 # Provide a dummy environment for nix, so that it will not access files outside the macOS sandbox.
 # Set cores to 0 because otherwise nix show-config resolves the cores based on the current machine
 dummy-env = env -i \
-	HOME=/dummy \
-	NIX_CONF_DIR=/dummy \
-	NIX_SSL_CERT_FILE=/dummy/no-ca-bundle.crt \
-	NIX_STATE_DIR=/dummy \
-	NIX_CONFIG='cores = 0'
+  HOME=/dummy \
+  NIX_CONF_DIR=/dummy \
+  NIX_SSL_CERT_FILE=/dummy/no-ca-bundle.crt \
+  NIX_STATE_DIR=/dummy \
+  NIX_CONFIG='cores = 0'
 
 nix-eval = $(dummy-env) $(bindir)/nix eval --experimental-features nix-command -I nix/corepkgs=corepkgs --store dummy:// --impure --raw
 
+$(d)/nix-env-%.1: $(d)/src/command-ref/nix-env/%.md
+  @printf "Title: %s\n\n" "$(subst nix-env-,nix-env --,$$(basename "$@" .1))" > $^.tmp
+  $(render-subcommand)
```

Comment:
Make needs this to be a tab.

Reply:
I think you did tabs -> spaces in this whole file which do don't want for make files.

Reply:
It was mixed to begin with, for some reason. Why tabs not spaces?

Reply:
https://retrocomputing.stackexchange.com/questions/20292/why-does-make-only-accept-tab-indentation

Reply:
https://www.gnu.org/software/make/manual/html_node/Rule-Introduction.html

Reply:
Will fix that. Apparently my Make foo got a bit dusty.